config.py

```diff
@@ -156,7 +156,7 @@ class configuration:
   compiler_flags = dict()
 
   compiler_flags['gcc'] = ['-c', '-O3', '-ffast-math', '-fopenmp',
-                           '-std=c++11', '-fpic']
+                           '-std=c++11', '-fpic', '-Wfloat-equal']
```

Comment:
This makes installing openmoc output a lot of warnings, for Universe.cpp, TrackGenerator.cpp and Geometry.cpp. 
I would rather you delete that flag from the regular user installation process.
Alternatively, fix all the causes of warnings, but it's not really worth the time, these are not bugs.